nsqadmin/logger.go

```diff
@@ -1,5 +1,57 @@
 package nsqadmin
 
+import (
+	"fmt"
+	"strings"
+)
+
 type Logger interface {
 	Output(maxdepth int, s string) error
 }
+
+const (
+	LOG_DEBUG = 1
+	LOG_INFO  = 2
+	LOG_WARN  = 3
+	LOG_ERROR = 4
+	LOG_FATAL = 5
+)
+
+func (n *NSQAdmin) logLevelFromString(level string) int {
+	// check log-level is valid and translate to int
+	switch strings.ToLower(level) {
+	case "debug":
+		return LOG_DEBUG
+	case "info":
+		return LOG_INFO
+	case "warn":
+		return LOG_WARN
+	case "error":
+		return LOG_ERROR
+	case "fatal":
+		return LOG_FATAL
+	default:
+		return -1
```

Comment:
(This can be fixed in a follow up).

This feels like a C idiom, not Go. If this function has error conditions, it should return `(int, error)` and this should be a case that returns an error.